extensions/libraries/redcore/api/api.php

```diff
@@ -261,10 +261,12 @@ public static function getPostedData()
 			$inputData = $input->post->getArray();
 		}
 
-		// Filter data with JInput default filter
-		$postedData = new JInput($inputData);
+		$filter = JFilterInput::getInstance(array(), array(), 1, 1, 1, -1);
```

Comment:
probably because in latest version it is this https://github.com/joomla/joomla-cms/blob/3.4.x/libraries/joomla/filter/input.php#L153 :)
but if it is in staging it will be in later versions...